README.md

```diff
@@ -0,0 +1,41 @@
+# Grist-Pack: an all-in-one packaging of Grist for cloud providers
+
+This is a [Packer](https://developer.hashicorp.com/packer/docs/intro) build of Grist that generates virtual machine images for cloud providers. Currently Amazon Web Services and DigitalOcean are supported. The generated images will have a `grist` user that is mostly setup to run Grist via [Docker Compose](https://docs.docker.com/compose/).
+
+
+## Building the virtual machine images
+
+Follow these steps to build AMIs for AWS and Snapshots for DigitalOcean (DO).
+
+1. [Install Packer](https://developer.hashicorp.com/packer/install).
+2. clone this repository: `git clone https://github.com/gristlabs/grist-pack.git`
+3. Initialise packer: `cd grist-pack && packer init .`
+
+You now need to obtain credentials for AWS and DO.
+
+1. For AWS, you need to obtain an access key with its corresponding secret.
+2. For DO, you will need a token.
+
+Create a file called `grist.auto.pkrvars.hcl` and put those secrets into it. For example,
```

Comment:
This was all clear and worked fine. In a future iteration, a section on minimal rights that packer will need would be helpful. But not now, this is fine for now.